FreeRTOS_IP.c

```diff
@@ -2961,6 +3134,19 @@ static eFrameProcessingResult_t prvProcessIPPacket( IPPacket_t * pxIPPacket,
             eReturn = prvCheckIP4HeaderOptions( pxNetworkBuffer );
         }
 
+        #if ( ipconfigUSE_IPv6 != 0 )
+            if( ( pxIPPacket->xEthernetHeader.usFrameType == ipIPv6_FRAME_TYPE ) &&
+                ( xGetExtensionOrder( ucProtocol, 0U ) > 0 ) )
+            {
+                eReturn = eHandleIPv6ExtensionHeaders( pxNetworkBuffer, pdTRUE );
```

Comment:
Maybe we need a config parameter since we are dealing with `memmove` etc. here.
It is similar to IP options which we may or may not accept based on the config value. What do you think about `ipconfigALLOW_IPv6_WITH_EXTENSIONS_HEADERS`?

Reply:
In an earlier version there was such a config macro, but then I realised that we must process packets that carry an extension header.